doc/12_building_and_artifacts.md

```diff
@@ -36,7 +36,7 @@ A fat jar with `assembly` suffix will be generated to:
 
     spark-cassandra-connector/connector/target/scala-{binary.version}/
 
-The jar contains Spark Cassandra Connector and its dependencies. Some of the dependencies are shaded to avoid 
+The jar contains Spark Cassandra Connector and it's dependencies. Some of the dependencies are shaded to avoid 
```

Comment:
nit.    "The jar contains *the* Spark Cassandra Connector and *its* dependencies".  "It’s" is a contraction of “it is” or “it has.” "Its" is a possessive determiner used to say that something belongs to or refers to something.  Most of the time apostrophe-s does signify a possessive but not in this case.